### ISSUE TITLE
Support for Azure Security Contacts

### ISSUE THREAD

**Issue by jsonpr** [cloudquery/cloudquery#6659]
Customer reported `azure_security_contacts` is not found.

https://learn.microsoft.com/en-us/rest/api/defenderforcloud/security-contacts

**Comment by jsonpr** [comment]
Should be addressed by https://github.com/cloudquery/cloudquery/pull/6374

**Comment by jsonpr** [comment]
Microsoft API has an issue: https://github.com/Azure/azure-sdk-for-go/issues/19740 - blocked by this.

**Comment by erezrokah** [comment]
I'm going to close this as `wontfix` for now, as we can't really do anything about it until Azure fixes the issue.
Once they do we should get the new version automatically (renovate should take care of it).
I'm also watching https://github.com/Azure/azure-sdk-for-go/issues/19740 for updates

**Comment by bbernays** [comment]
> Microsoft API has an issue: [Azure/azure-sdk-for-go#19740](https://github.com/Azure/azure-sdk-for-go/issues/19740) - blocked by this.

For others that run into this issue. The error that is thrown is:
```
unmarshalling type armsecurity.ContactList: unmarshalling typearmsecurity.ContactList: json: cannot unmarshal array into Go value of type map[string]json.RawMessage
```

or
```
unmarshalling type *armsecurity.ContactList: unmarshalling type *armsecurity.ContactList: json: cannot unmarshal array into Go value of type map[string]json.RawMessage
```
